ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/HolderAccess.java

```diff
@@ -0,0 +1,5 @@
+package de.adorsys.ledgers.baam.db.domain;
+
+public class HolderAccess {
+
+}
```

Comment:
Is this all about the HolderAccess class, or you are still to push the updated code?